Removing from imgsz too, it's not working with input lists:

>       checks.check_imgsz([600, 600], max_dim=1)
E       TypeError: unhashable type: 'list'

--- a/ultralytics/utils/checks.py
+++ b/ultralytics/utils/checks.py
@@ -344,9 +344,8 @@ def check_python(minimum: str = "3.8.0", hard: bool = True, verbose: bool = Fals
     """
     return check_version(PYTHON_VERSION, minimum, name="Python", hard=hard, verbose=verbose)
 
-
-@TryExcept()
 @functools.lru_cache
+@TryExcept()
 def check_requirements(requirements=ROOT.parent / "requirements.txt", exclude=(), install=True, cmds=""):
     """
     Check if installed dependencies meet Ultralytics YOLO models requirements and attempt to auto-update if needed.

--- a/ultralytics/utils/checks.py
+++ b/ultralytics/utils/checks.py
@@ -111,7 +111,6 @@ def is_ascii(s) -> bool:
     return all(ord(c) < 128 for c in str(s))
 
 
-@functools.lru_cache
 def check_imgsz(imgsz, stride=32, min_dim=1, max_dim=2, floor=0):
     """
     Verify image size is a multiple of the given stride in each dimension. If the image size is not a multiple of the

--- a/ultralytics/utils/checks.py
+++ b/ultralytics/utils/checks.py
@@ -344,6 +344,7 @@ def check_python(minimum: str = "3.8.0", hard: bool = True, verbose: bool = Fals
     """
     return check_version(PYTHON_VERSION, minimum, name="Python", hard=hard, verbose=verbose)
 
+
 @functools.lru_cache
 @TryExcept()
 def check_requirements(requirements=ROOT.parent / "requirements.txt", exclude=(), install=True, cmds=""):

